Scripts/ClearBinAndObjDirs.bat

```diff
@@ -0,0 +1,8 @@
+@echo off
+
+cd ..
+
+for /f "tokens=*" %%f in ('dir ".\" /a:d /b') do (
+	rmdir /q /s "%%f\bin" > nul 2> nul
+	rmdir /q /s "%%f\obj" > nul 2> nul
+	)
```

Comment:
reduce the intent of `)`

```suggestion
)
```

Reply:
Done by da8ca90d1859ae0049fa8b153930ff123c61b082.